### ISSUE TITLE
Docs / Examples avalible?

### ISSUE THREAD

**Issue by hrueger** [streamlabs/obs-studio-node#578]
Is there any documentation or examples avalible? I'd like to use this in an electron project. I had a look at the TypeScript definitions and at the Stream-labs obs code, but since this looks very complicated to a developer without vue experience, I couldn't even figure out how to mix two sources together and get the result.

**Comment by hrueger** [comment]
Just a little example how to create a stream out of two sources and preview it and I'd be happy to create a pull request with a more detailed documentation after I've figured that out.
But I'd need a starting point...

**Comment by JonathanTreffler** [comment]
First of all you need to build this into a nodejs package by following [the instructions](https://github.com/stream-labs/obs-studio-node#building)

This is where the package exposes it's c++ code to the js api:
https://github.com/stream-labs/obs-studio-node/blob/staging/js/module.ts
So every Object defined there will be acessible in your js code, if you import the module.
It might be helpful to look at the test code in [/tests/](https://github.com/stream-labs/obs-studio-node/tree/staging/tests).
(I cannot guarantee that my information here are correct, because i never used obs-studio-node. I am just assuming based on the code)

**Comment by emptycrown** [comment]
@hrueger Did you ever get an example app working with node? Would love to see a simple example if possible -- thanks a bunch! Even just seeing the basic layout could save me a lot of time.

**Comment by hrueger** [comment]
@EmptyCrown No, unfortunately, I couldn't. I'm working on using [obs-websocket-js](https://github.com/haganbmj/obs-websocket-js) to control obs. If you can make it work with `obs-studio-node`, I'd love to see that!

**Comment by Envek** [comment]
We, together with @EmptyCrown, have built minimal example application that uses obs-studio-node to record screen capture:

https://github.com/Envek/obs-studio-node-example

I tried to keep the code as simple and straightforward as possible to have only essential things there.

**Comment by EddyGharbi** [comment]
Thanks for doing that, it looks very good. If you decide to share this tutorial through the official repository, it would be a pleasure for me to look at it.

**Comment by hrueger** [comment]
I'm working on this. Can I move the building instructions into  `BUILDING.md` and put the docs in the `README.md`? Or do you prefer a `DOCS.md`?
